.github/workflows/rave.yml

```diff
@@ -50,19 +50,25 @@ jobs:
           # Map each version and if the number is in major.minor syntax, append a .0
           versions=$(jq -c '. | map(.version | capture("(?<major>\\d+)\\.(?<minor>\\d+)(\\.(?<patch>\\d+))?") | .major + "." + .minor + "." + (.patch // "0") )' offers.json)
           echo "$versions" > versions.json
+
       - name: Fetch version matrix from workflow input
         if: ${{ github.event_name == 'workflow_dispatch' }}
         run: | #shell
-          echo '["${{ inputs.version }}"]' > versions.json
+          echo "[\"${VERSION}\"]" > versions.json
```

Comment:
This is really
`jq --null-input --compact-output --arg version "$VERSION" '[$version]'`

Reply:
Oh that looks better, thanks.